src/pages/en/analyse/index.html

```diff
@@ -0,0 +1,15 @@
+---
+lang_en: true
+page-title: Open-Source Project Corona-Warn-App - Analysis
+page-description:
+page-name: analyse
+---
+
+<div class="" style="background: #D4EAF7; padding: 16px; font-size: 14px; line-height: 30px;">
+	<div class="container">
+		Currently, the Corona Warning App (CWA) Key Figures are available in German only. We are working on the translated version which will be online soon. 
```

Comment:
"Corona-Warn-App" is "Corona-Warn-App" in English and not "Corona Warning App".

```suggestion
		Currently, the Corona-Warn-App (CWA) Key Figures are available in German only. We are working on the translated version which will be online soon. 
```